src/Raven.Server/Documents/CollectionRunner.cs

```diff
@@ -190,11 +190,11 @@ protected long GetTotalCountForCollection(DocumentsOperationContext context, str
             if (isAllDocs)
             {
                 var allDocsCount = Database.DocumentsStorage.GetNumberOfDocuments(context);
-                Database.DocumentsStorage.GetNumberOfDocumentsToProcess(context, CollectionName.HiLoCollection, 0, out long hiloDocsCount);
+                Database.DocumentsStorage.GetNumberOfDocumentsToProcess(context, CollectionName.HiLoCollection, 0, out long hiloDocsCount, Stopwatch.StartNew());
                 return allDocsCount - hiloDocsCount;
             }
 
-            Database.DocumentsStorage.GetNumberOfDocumentsToProcess(context, collectionName, 0, out long totalCount);
+            Database.DocumentsStorage.GetNumberOfDocumentsToProcess(context, collectionName, 0, out long totalCount, Stopwatch.StartNew());
```

Comment:
Why do we need to pass `Stopwatch.StartNew()`? It looks a bit suspicious

Reply:
We need to ensure that we aren't spending too long here.
That measures that total time that we spend waiting to run this computation